app/controllers/application_controller.rb

```diff
@@ -9,6 +9,8 @@ class ApplicationController < ActionController::Base
 
   private
 
+  def current_profile_user
+
```

Comment:
Remove these 2 lines and test would pass, as currently missing end would be generating error.